File: pset1/mario.c
// to create a pyramid

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    while (true)
    {
        height = get_int("Enter the height :\t");

// checking input        
        if (height >= 1 && height <= 8)
        {
            break;
        }
    }

//pyramid building
    for (int i = 0; i < height; ++i)
    {
//left pyramid's space 
        for (int spc = height - i - 1; spc > 0 ; --spc)
        {
            printf(" ");
        }
//left pyramid
        for (int j = 0; j <= i; ++j)
        {
            printf("#");            
        }
        printf("  ");
//right pyramid
        for (int j = 0; j <= i; ++j)
        {
            printf("#");  
        }
        printf("\n");    
    }
}

File: python/pset6/mario/more/mario.py
# to create a pyramid in py

from cs50 import get_int

# checking input
height = 0
while (height > 8 or height < 1):
    height = get_int("Height:\n")

#pyramid building
for i in range(1, height + 1):
    print(" " * (height - i), end="")
    print("#" * i, end="  ")
    print("#" * i)